rally.yml

```diff
@@ -15,7 +15,7 @@ checkCommitMessages: true
 mergeOnPRBody: true
 
 # Comment on the PR in addition to the check message? (true | false)
-commentOnPull: false
+commentOnPull: true
```

Comment:
@primetheus Should this default be true or would you rather it default to false?

Reply:
@zkoppert this should be false by default. The reason it's present is because older versions of GHES don't have the `Checks API` and there won't be a place for these results to be posted. In those cases, users can opt for having the results posted as comments to the PR. I don't think that's the desired default though

Reply:
That makes sense to me.